wrapper.js

```diff
@@ -27,6 +27,11 @@ function setupMethods (soljson) {
 
   // This calls compile() with args || cb
   var runWithReadCallback = function (readCallback, compile, args) {
+    // Forward compatibility with 0.6.x
+    if (typeof readCallback === 'object') {
+      readCallback = readCallback.imports;
```

Comment:
@leonardoalt @chriseth what should be the name of the callback?

Reply:
I'm fine with anything reasonable similar to that.
import, imports, read